### PR TITLE
Adds launch files to debug in rviz

### PR DESCRIPTION
**Pull request by raabuchanan**
Adds a launch file to visualize the urdf in rviz for quick debugging

Screenshot from 2023-05-02 21-18-44

### CODE REVIEW

File: nextagea_description/launch/view_urdf.launch [hunk 1]
```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0"?>
+<launch>
+  <!-- Name of the robot description parameter -->
+  <arg name="description_name"         default="robot_description"/>
+  <!-- Set robot description path -->
+  <!-- <arg name="description_file"         default="$(find anymal_b_simple_description)/urdf/anymal.urdf"/> -->
```
Comment (joaomoura24) [inline nextagea_description/launch/view_urdf.launch:6]: Even though this is commented, I would change it to the nextage urdf path:
"$(find nextagea_description)/urdf/nextagea.urdf"

**Comment by joaomoura24** [comment]
Good addition. Thank you. I tested it and it works well.
Check the comment and then i can merge.

**Comment by raabuchanan** [comment]
Sorry, did you make a comment somewhere? I can't see any

**Review by joaomoura24 — CHANGES_REQUESTED** [review]

File: nextagea_description/launch/view_urdf.launch (hunk 1 above)
Comment (joaomoura24) [reply nextagea_description/launch/view_urdf.launch:6]: Sorry, forgot to submit the review.

**Comment by raabuchanan** [comment]
Done

**Review by joaomoura24 — APPROVED** [review]

**Pull request merged**